src/main/resources/application.properties

```diff
@@ -40,6 +40,7 @@ job.repository.password=
 ## true: The already COMPLETEd steps will be rerun. This is restarting the job from the beginning
 ## false(default): if the job was aborted and is relaunched, COMPLETEd steps will NOT be done again
 config.restartability.allow=false
+config.chunk.size=
```

Comment:
The default is in the `JobOptions` class, as many of the other options in this `.properties`. I did a quick test and if the default is not the `JobOptions` class, the `ApplicationContext` failed to start.